src/dlgProfilePreferences.cpp

```diff
@@ -2911,21 +2909,18 @@ void dlgProfilePreferences::slot_changeShowLineFeedsAndParagraphs(const bool sta
 
 /*
  * This is to deal particularly with the case where the preferences dialog is
- * opened without a host instance (other than the dummy "default_host") being
- * around - and then the user starts up a profile and one gets created.
+ * opened without a host instance being around - and then the user starts up
+ * a profile and one gets created.
  * In that situation we detect the signal that the mudlet class sends out (now)
  * when a host is created and wire it up into the controls that until then
  * have been disabled/greyed-out.
  */
 void dlgProfilePreferences::slot_handleHostAddition(Host* pHost, const quint8 count)
 {
-    // count will be 2 in the case we particularly want to handle (adding the
-    // first real Host instance):
-    if (!mpHost && pHost && count < 3) {
+    if (!mpHost && pHost && count < 1) {
```

Comment:
There were <3 before, you remove default_host, now it should be <2, right?

Reply:
Thanks, fixed.